### DISCUSSION TITLE
CSC Compiler Crash (likely RE Collection Expressions)

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/csharplang#8041] [General]
Cross posting this here for visibility:
https://github.com/dotnet/roslyn/issues/72903

**Comment by 333fred** [comment]
Tony, please don't cross-post here. This is not the correct forum for bug reports, as your clearly know, and it's not more likely to get the right people to look at your issue (particularly on the weekend!) There are a _lot_ of people who don't work at Microsoft and aren't interested in bug reports subscribed to this repo, and creating this discussion creates noise for all of them (whether that's an email, github notification, or something else). Your report will be triaged as all other bug reports are triaged, thanks.

**Reply by TonyValenti** [reply]
Sorry @333fred !  You're right!
